docs/modules/con-tssc-opinions.adoc

```diff
@@ -0,0 +1,16 @@
+[id="{ProjectNameID}-opinions", reftext="{ProjectShortName} Opinions"]
+= {ProjectShortName} Opinions
+
+The {ProjectName} (ProjectShortName) is an opinionated software development and deployment workflow. The key being, opinionated. Which means it is filled with opinions, some direct, some less so. This document is meant to capture all of those opinions which feed into the design and implementation of the TSSC.
```

Comment:
Still need to update the last TSSC var. @itewk

Reply:
fixed.